pymc/tests/test_sampling.py

```diff
@@ -1042,6 +1042,28 @@ def point_list_arg_bug_fixture() -> Tuple[pm.Model, pm.backends.base.MultiTrace]
 
 
 class TestSamplePriorPredictive(SeededTest):
+    def test_idata_output(self):
```

Comment:
I think this test should be in idata_conversion. And I would try to avoid having a new test but instead checking for constant data in existing tests where it is currently missing